README.md

```diff
@@ -141,8 +141,8 @@ az vm image terms accept --offer graphdb-ee --plan graphdb-byol --publisher onto
 | gateway\_probe\_interval | Interval in seconds between the health probe checks | `number` | `10` | no |
 | gateway\_probe\_timeout | Timeout in seconds for the health probe checks | `number` | `1` | no |
 | gateway\_probe\_threshold | Number of consecutive health checks to consider the probe passing or failing | `number` | `2` | no |
-| tls\_certificate\_path | Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. | `string` | `null` | no |
-| tls\_certificate\_password | TLS certificate password for password protected certificates. | `string` | `null` | no |
+| tls\_certificate\_path | Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. | `string` | n/a | yes |
```

Comment:
| Property                  | Description                                                                                                                                                    | Type     | Default | Required |
|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|----------|
| `tls_certificate_path`     | Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. **Either `tls_certificate_path` or `tls_certificate_password` must be provided.** | `string` | n/a     | yes      |
| `tls_certificate_password` | TLS certificate password for password-protected certificates. **Either `tls_certificate_password` or `tls_certificate_path` must be provided.**                  | `string` | n/a     | yes      |